offer-manager-domain/src/test/groovy/life/qbic/business/offers/TaxOfficeSpec.groovy

```diff
@@ -19,6 +20,22 @@ class TaxOfficeSpec extends Specification{
     Affiliation externalAcademicAffiliation
     @Shared
     Affiliation externalAffiliation
+    @Shared
+    Product primaryAnalysis = new PrimaryAnalysis("", "", 0, ProductUnit.PER_FLOW_CELL, "1")
+    @Shared
+    Product secondaryAnalysis = new SecondaryAnalysis("", "", 0, ProductUnit.PER_FLOW_CELL, "1")
+    @Shared
+    Product sequencing = new Sequencing("", "", 0, ProductUnit.PER_FLOW_CELL, "1")
+    @Shared
+    Product proteomicAnalysis = new ProteomicAnalysis("", "", 0, ProductUnit.PER_FLOW_CELL, "1")
+    @Shared
+    Product metabolomicAnalysis = new MetabolomicAnalysis("", "", 0, ProductUnit.PER_FLOW_CELL, "1")
+    @Shared
+    Product dataStorage= new DataStorage("", "", 0, ProductUnit.PER_FLOW_CELL, "1")
+    @Shared
+    Product projectManagement = new ProjectManagement("", "", 0, ProductUnit.PER_FLOW_CELL, "1")
```

Comment:
NIT
You could use here the same constructor as below (with long for running number and with the facility)